setup-scripts/blocklist-airtable.py

```diff
@@ -122,70 +122,36 @@ async def block_skylinks_from_airtable():
         )
         await send_msg(message, file=("\n".join(invalid_skylinks)))
 
-    apipassword = exec("docker exec sia cat /sia-data/apipassword")
     ipaddress = exec(
-        "docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' sia"
+        "docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nginx"
     )
 
-    print("Sending blocklist request to siad")
+    print("Sending blocklist request to siad through nginx")
     response = requests.post(
-        "http://" + ipaddress + ":9980/skynet/blocklist",
-        auth=("", apipassword),
-        headers={"user-agent": "Sia-Agent"},
+        "http://" + ipaddress + ":8000/skynet/blocklist",
         data=json.dumps({"add": skylinks}),
     )
 
-    if response.status_code == 204:
-        print("Siad blocklist successfully updated with provided skylink")
-    else:
+    print(json.dumps({"add": skylinks}))
+
+    if response.status_code != 204:
         status_code = str(response.status_code)
         response_text = response.text or "empty response"
         message = (
-            "Siad blocklist endpoint responded with code "
+            "Airtable blocklist request responded with code "
             + status_code
             + ": "
             + response_text
         )
         return await send_msg(message, force_notify=False)
 
-    # Remove from NGINX cache
-    # NOTE:
```

Comment:
Should do the same for the `blocklist-skylink.sh` script

Reply:
> Should do the same for the `blocklist-skylink.sh` script

Changes in https://github.com/SkynetLabs/skynet-webportal/pull/1497/commits/dac4d57b1236a6250ce21ae782a3cd63a26dd643